pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -23,16 +23,20 @@ mkDerivation rec {
   buildInputs = [
     qtbase qtimageformats ffmpeg_3 openalSoft lzma lz4 xxHash libdbusmenu
     zlib minizip openssl hunspell libtgvoip microsoft_gsl tl-expected range-v3
+    libopus alsaLib libpulseaudio
   ];
 
   qtWrapperArgs = [
     "--prefix PATH : ${xdg-utils}/bin"
   ];
 
   cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
Isn't it automatically set by Nix?

Reply:
Looking at other derivations, doesn't seem so.